src/components/ConsoleEntry.jsx

```diff
@@ -3,17 +3,28 @@ import PropTypes from 'prop-types';
 import {ConsoleEntry as ConsoleEntryRecord} from '../records';
 import ConsoleOutput from './ConsoleOutput';
 
-export default function ConsoleEntry({entry}) {
-  const {expression} = entry;
+export default function ConsoleEntry({entry, currentCompiledProjectKey}) {
+  const {expression, evaluatedByCompiledProjectKey} = entry;
+  const isActive = currentCompiledProjectKey === evaluatedByCompiledProjectKey;
 
   return (
     <div className="console__entry">
-      <div className="console__expression">{expression}</div>
-      <ConsoleOutput entry={entry} />
+      <div
+        className={
+          isActive ? 'console__expression' : 'console__expression_inactive'
+        }
+      >
+        {expression}
+      </div>
+      <ConsoleOutput
+        currentCompiledProjectKey={currentCompiledProjectKey}
```

Comment:
I would probably just pass `isActive` as a prop, no need to make the `<ConsoleOutput>` do the same logic again

Reply:
That makes sense